products/mcp/pnpm-lock.yaml

```diff
@@ -1,525 +0,0 @@
-lockfileVersion: '9.0'
```

Comment:
this file didn't do anything so I've deleted it, we're in a pnpm workspace so the one at the root is the one that matters